username username
string username_sig
string password
string pass_sig
---
bool success
